mne/epochs.py

```diff
@@ -1233,6 +1242,16 @@ def __getitem__(self, key):
             key = [key]
 
         if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):
+            if any(["/" in k_i for k_i in epochs.event_id.keys()]):
```

Comment:
Probably because our Epochs subclassing hierarchy is a mess. At some point we need to reconcile what goes in `_BaseEpochs` vs `Epochs`. I started trying to move everything to `_BaseEpochs` at one point, and then realized I didn't know what all I could move without breaking `RtEpochs`. So @mainakjas you might actually be the best candidate to move everything over since you know what will break `RtEpochs`.